### DISCUSSION TITLE
Global/local version difference

### DISCUSSION THREAD

**Discussion by fishuke** [vercel/next.js#37173] [Ideas]
### Describe the feature you'd like to request

Give a warning that says "Your global and local page version is different please consider updating."

### Describe the solution you'd like

With this solution users will not get some weird errors that the reason can not be found.

### Describe alternatives you've considered

Add a warning on errors.

**Comment by timneutkens** [comment]
It's unclear what you mean with "global and local page version".

**Reply by fishuke** [reply]
Globally installed package and local package.

**Reply by GabenGar** [reply]
Local packages take precedence over the global ones so if you are iffy about versions then just install the package locally.

**Reply by fishuke** [reply]
> Local packages take precedence over the global ones so if you are iffy about versions then just install the package locally.

I had problem on windows global package takes precedence on local version.

**Reply by GabenGar** [reply]
Try `npm install` in your project folder.

**Reply by fishuke** [reply]
You can try on a windows device if this errors occurs. I am pretty sure i did yarn install.
